src/main/scala/io/pubcrawler/stout/db/Schemas.scala

```diff
@@ -0,0 +1,82 @@
+package io.pubcrawler.stout.db
+
+import slick.jdbc.PostgresProfile.api._
+import java.sql.Date
+import java.sql.Time
+
+
+class Stop(tag: Tag) extends Table[(Int, String, String, Float, Float)](tag, "STOP") {
```

Comment:
I think the `Table` subclasses should have Plural naming, which would be `Stops` in this case.

Any chance we can throw in some case classes to represent instances? http://slick.lightbend.com/doc/3.1.1/schemas.html#mapped-tables